tests/e2e/sidecar/sidecar-pod/02-assert.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  labels:
+    app: vertx-create-span-sidecar
+spec:
+  containers:
+    - name: vertx-create-span-sidecar
```

Comment:
The test is failing because the image is missing here, I think

Reply:
mh.. i wonder because all the other test dont provide images :mag: